### ISSUE TITLE
Color contrast between tick and background is 2.422:1 which is less than 3:1 under 'Query History' section: A11y_mssql for VSCode_mssql Extension_Query History_Non-Text Contrast

### ISSUE THREAD

**Issue by aniketr99** [microsoft/vscode-mssql#17878]
## GitHubTags:
#A11yTCS;#Win11;#DesktopApp;#BM_mssql for VSCode_Win32_Feb2024;#A11ySev2;#mssql for VSCode;#WCAG1.4.11;#A11yMAS;#Win32;#Non-Text Contrast;#A11yWCAG2.1;

## Environment Details:
Application Name: mssql for VSCode
Operating System: Win11[23H2]
OS Build: 26016.1012
Visual studio code: 1.86.2
Extension Version: v1.22.1

## Repro Steps:

1. Open Visual studio code 
2. Click on extension button and search for "mssql"
3. Click on install and move to the SQL Server  button
4. Click on Add Connection Button and insert the valid credentials
5. Right click on the server/database node you want to run the new query on.
6. Click on “New Query” write any valid query and click on "Execute query button"
7. in the left panel Query History window will appears
8. Open Accessibility insights for windows and verify whether color contrast for tick and background is less than 3:1
 or not.

## Actual:
Color contrast between tick and background is 2.422:1 which is less than 3:1 under 'Query History' section.

## Expected:
Color contrast between tick and background should be greater than or equal to 3:1 under 'Query History' section.

## User Impact:

Low vision users will not be able to see the tick icon clearly.

[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

## Attachment:
![Screenshot (376)](https://github.com/microsoft/vscode-mssql/assets/100269580/7d0a59fa-a313-465c-8264-1e6c19c49dbb)

**Comment by laurennat** [comment]
Fixed in PR #17928 : https://github.com/microsoft/vscode-mssql/pull/17928
